src/sentry/static/sentry/app/components/smartSearchBar/index.jsx

```diff
@@ -446,6 +447,38 @@ class SmartSearchBar extends React.Component {
     ];
   };
 
+  getReleases = _.debounce(
+    async (tag, query) => {
+      const releasePromise = this.fetchReleases(query);
+
+      const tags = this.getPredefinedTagValues(tag, query);
+      const tagValues = tags.map(v => ({
+        ...v,
+        type: 'first-release',
+      }));
+
+      const releases = await releasePromise;
+      const releaseValues = releases.map(r => ({
+        value: r.shortVersion,
+        desc: r.shortVersion,
+        type: 'first-release',
+      }));
+
+      return [...tagValues, ...releaseValues];
+    },
+    300,
+    {leading: true}
+  );
+
+  fetchReleases = async query => {
+    const {api, organization} = this.props;
+
+    const queryString = new URLSearchParams(window.location.search);
```

Comment:
Fixed: I was using `window.location.search` to avoid prop-drilling but I found that the `router` prop is passed down using `Context` so I have switched to that.